src/Microsoft.AspNet.Hosting/WebHostBuilder.cs

```diff
@@ -91,6 +92,10 @@ private IServiceCollection BuildHostingServices()
             services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
             services.AddLogging();
 
+            var telemetrySource = new TelemetryListener("Microsoft.AspNet");
+            services.AddInstance<TelemetrySource>(telemetrySource);
+            services.AddInstance<TelemetryListener>(telemetrySource);
```

Comment:
Orthogonal to this PR, but @davidfowl - do you still think we need to do something different here?